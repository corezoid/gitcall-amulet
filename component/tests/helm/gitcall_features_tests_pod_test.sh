#!/usr/bin/env bash
$CZ helm template | sed -n '/gitcall-features-tests-pod\.yaml/,/Source\:/p'
