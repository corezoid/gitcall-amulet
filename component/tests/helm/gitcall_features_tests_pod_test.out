# Source: gitcall_amulet_release/charts/gitcall_amulet/templates/gitcall-features-tests-pod.yaml
apiVersion: batch/v1
kind: Job
metadata:
  name: RELEASE-NAME-gitcall-amulet-features-tests-job
  annotations:
    "helm.sh/hook": test
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "gitcall_amulet"
spec:
  activeDeadlineSeconds: 180 # 3min
  ttlSecondsAfterFinished: 360 # 6min
  backoffLimit: 0
  template:
    metadata:
      labels:
        release: "RELEASE-NAME"
        app: "gitcall"
        tier: "gitcall_amulet"
    spec:
      restartPolicy: Never
      containers:
        - name: main
          imagePullPolicy: "IfNotPresent"
          image: docker-hub.middleware.biz/gitcall/gitcall-amulet:aVersion
          command: ["/app/gitcall_features_tests"]
          args:
            - "--service-id=deploy-test"
            - "-test.v"
            - "--gitcall-amqp=amqp://rmqUser:rmqPass@rmq.host:5672/gitcall"
            - "--dundergitcall-amqp=amqp://rmqUser:rmqPass@rmq.host:5672/dundergitcall"
          resources:
            limits:
              cpu: 100m
              memory: 50Mi
            requests:
              cpu: 100m
              memory: 50Mi
      imagePullSecrets:
      - name: "the-image-pull-secret"
---
# Source: gitcall_amulet_release/charts/gitcall_amulet/templates/gitcall-network-policy-tests-pod.yaml
